examples/simple-authenticated-api/lib/simple-authenticated-api-stack.ts

```diff
@@ -23,7 +23,7 @@ export class SimpleAuthenticatedApiStack extends cdk.Stack {
     const alarmTopic = new sns.Topic(
       this,
       `${prefix}simple-lambda-Worker-alarm`,
-      { topicName: `${prefix}simple-lambda-worker-alarm` }
+      { topicName: `${prefix}simple-authenticated-api-alarm` }
```

Comment:
Correcting a copy-paste error. This stops both example stacks being deployed at the same time because the resource already exists in whichever stack was deployed first.